diff --git a/node_modules/wagmi/dist/wagmi.cjs.dev.js b/node_modules/wagmi/dist/wagmi.cjs.dev.js
index 909d868..8b79cf8 100644
--- a/node_modules/wagmi/dist/wagmi.cjs.dev.js
+++ b/node_modules/wagmi/dist/wagmi.cjs.dev.js
@@ -8,7 +8,7 @@ var reactQuery = require('react-query');
 var persistQueryClient = require('react-query/persistQueryClient');
 var createWebStoragePersister = require('react-query/createWebStoragePersister');
 var ethers = require('ethers/lib/ethers');
-var pkg = require('use-sync-external-store/shim/index.js');
+var pkg = require('use-sync-external-store/shim');
 var withSelector_js = require('use-sync-external-store/shim/with-selector.js');
 
 function _interopNamespace(e) {
diff --git a/node_modules/wagmi/dist/wagmi.cjs.prod.js b/node_modules/wagmi/dist/wagmi.cjs.prod.js
index 909d868..8b79cf8 100644
--- a/node_modules/wagmi/dist/wagmi.cjs.prod.js
+++ b/node_modules/wagmi/dist/wagmi.cjs.prod.js
@@ -8,7 +8,7 @@ var reactQuery = require('react-query');
 var persistQueryClient = require('react-query/persistQueryClient');
 var createWebStoragePersister = require('react-query/createWebStoragePersister');
 var ethers = require('ethers/lib/ethers');
-var pkg = require('use-sync-external-store/shim/index.js');
+var pkg = require('use-sync-external-store/shim');
 var withSelector_js = require('use-sync-external-store/shim/with-selector.js');
 
 function _interopNamespace(e) {
diff --git a/node_modules/wagmi/dist/wagmi.esm.js b/node_modules/wagmi/dist/wagmi.esm.js
index eb3e237..892e51e 100644
--- a/node_modules/wagmi/dist/wagmi.esm.js
+++ b/node_modules/wagmi/dist/wagmi.esm.js
@@ -5,7 +5,7 @@ import { QueryClient, QueryClientProvider, useQueryClient, useIsRestoring, useQu
 import { persistQueryClient } from 'react-query/persistQueryClient';
 import { createWebStoragePersister } from 'react-query/createWebStoragePersister';
 import { BigNumber, Contract } from 'ethers/lib/ethers';
-import * as pkg from 'use-sync-external-store/shim/index.js';
+import * as pkg from 'use-sync-external-store/shim';
 import { useSyncExternalStoreWithSelector } from 'use-sync-external-store/shim/with-selector.js';
 
 /** Forked from https://github.com/epoberezkin/fast-deep-equal */
